css/style.css

```diff
@@ -16,4 +15,125 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+header {
+  bottom: 20px;
+  top: 40px;
+}
+
+nav {
+  margin-top: 0;
+  position: relative;
+  z-index: 5;
+  height: 71;
+  bottom: 107;
+}
+
+ul {
+  display: flex;
+  list-style: none;
```

Comment:
it is okay that you used "display: flex"  for `ul` here because you just have one "ul", but generally, you should always create classes in your CSS file and  use `display: flex`  for the parent element and align its child element with other flex-box features